MSMetaEnhancer/app.py

```diff
@@ -60,6 +60,15 @@ def curate_spectra(self):
         """
         self.spectra = Curator().curate_spectra(self.spectra)
 
+    def get_service_conversions(self, service):
+        """
+        Method to get all conversions for given service.
+
+        :param service: given Converter subclass
+        :return: a list of available conversion functions
+        """
+        return service.get_conversions()
```

Comment:
I think this method could be omitted, as it doesn't even use `self` and simply forwards to `service.get_conversions()`.